text/0265-event-vision.md

```diff
@@ -0,0 +1,72 @@
+# Event basics
+
+## Motivation
+
+The introduction of Events has been contentious, so we want to write down and agree on a few basics.
+
+### What are events?
+
+Events are a type of log which has a required event name and a specific structure which is implied by that event name.
+
+The event name and its implied structure make events much more suitable for observability compared to traditional logs.
+
+Because of this, the OpenTelemetry project wants to encourage the transition from traditional logs to events.
+
+### OTLP
+
+Since events are a type of log, they share the same OTLP data structure and OTLP pipeline.
+
+### API
+
+OpenTelemetry should have an Event API. This will help to promote the distinction between events and traditional logs,
+and encourage the use of events over traditional logs.
+
+### Interoperability with traditional logs
+
+It should be possible to send events from the Event API to a traditional logging library (e.g. Log4j).
+This allows users to integrate events from the Event API into an existing (non-OpenTelemetry) log stream.
+
+Note: If a user chooses to send events from the Event API to a traditional logging library, and they have
+also chosen to send the logs from their traditional logging library to the OpenTelemetry Logging SDK, then they should
+avoid sending events from the Event API directly to the OpenTelemetry Logging SDK since that would lead to duplicate
+capture of events that were sent from the Event API.
+
+It should be possible to bypass the Event API entirely and emit Events directly via an existing language-specific logging libraries.
```

Comment:
Thanks for re-iterating this part.

Reply:
> It should be possible to bypass the Event API entirely and emit Events directly via an existing language-specific logging libraries.

I'd prefer to add some wording around this to highlight that this is a stop-gap and NOT the overall intention.
ie. that "today" it's technically possible to simulate an event via a log record, but instrumentations SHOULD not use this technique IF an Event API is available for the language.

Reply:
@MSNev I believe this is addressed below:

> OpenTelemetry will recommend that
[instrumentation libraries](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/glossary.md#instrumentation-library)
use the Event API over emitting events via a generic logging library in order to give users a simple and consistent
onboarding story that doesn't involve mixing the two approaches.
>
> OpenTelemetry will recommend that application developers also use the Event API over emitting events via a generic
logging library since it avoid accidentally emitting logs which lack an event name or are unstructured.